zanata-war/src/main/webapp/iteration/view.xhtml

```diff
@@ -127,10 +127,54 @@
     var uploadButton = getUploadSourceDocButton();
     uploadButton.prop('disabled', !enable);
   }
-
 </script>
 
+  <ui:fragment
+    rendered="#{versionHomeAction.documentRemovalAllowed}">
+    <zanata:confirm-modal id="deleteDocModal"
+      title="#{msgs['jsf.iteration.files.DeleteDocument']}"
+      message="#{msgs['jsf.iteration.files.ConfirmDocDeletion']}"
+      onresponse="onDeleteDocResponse"/>
+
+    <script type="application/javascript">
+      var selectedDoc;
+      function setSelectedDoc(id) {
+        selectedDoc = id;
+      }
+      function onDeleteDocResponse(del) {
+        if (del) {
+          deleteDoc(selectedDoc)
+        } else {
+          selectedDoc = null
+        }
+        zanata.modal.hide('#deleteDocModal');
+      }
+    </script>
+  </ui:fragment>
+
+  <ui:fragment rendered="#{identity.loggedIn}">
+    <script type="application/javascript">
+      function onExportAllTMXResponse(exp) {
+        if (exp) {
+          window.open("#{request.contextPath}/rest/tm/projects/#{versionHomeAction.projectSlug}/iterations/#{versionHomeAction.versionSlug}");
+        }
+        zanata.modal.hide('#exportTMXVersionModal');
+      }
+    </script>
+    <zanata:confirm-modal id="exportTMXVersionModal"
+      title="#{msgs['jsf.ExportTMXIter']}"
```

Comment:
Does not exist in messages.properties. There's a jsf.ExportTMX (which could potentially be renamed but I don't know what the impact would be)